### DISCUSSION TITLE
Announcing TiddlyGit 0.4.1

### DISCUSSION THREAD

**Discussion by linonetwo** [TiddlyWiki/TiddlyWiki5#5917] [General]
Download: https://github.com/tiddly-gittly/TiddlyGit-Desktop/releases/tag/v0.4.1

This is a desktop app that runs NodeJS TiddlyWiki, and it will create a git repo to backup your tiddlers.
NodeJS wiki save each tiddler into a separate file, so each tiddler will have its git history for undo and redo.

I fixed many bugs in this release, hope it works well this time...

**Comment by Jermolene** [comment]
Congratulations on the release @linonetwo, it's great to see your progress.
